client/aws/aws.go

```diff
@@ -62,6 +63,16 @@ func NewClients(config Config) Clients {
 	return clients
 }
 
+func NewCFClient(config Config) cloudformationiface.CloudFormationAPI {
```

Comment:
NewClients already returns a cloudformation client. Why do we need to use `cloudformation/cloudformationiface`?

Reply:
Indeed, this was a leftover, this interface is used as the type of one of the members of the Resource (in order to be able to unit test it with a mock, probably we can use a more ad-hoc interface, let's see), it doesn't belong here, fixed.